src/logic/config/model/networkConfig.ts

```diff
@@ -0,0 +1,67 @@
+import { ETHEREUM_NETWORK } from 'src/config/networks/network.d'
+
+import { NetworkSpecificConfiguration } from 'src/config'
+
+type GasPriceOracle = {
+  type: string
+  uri: string
+  gasParameter: string
+  gweiFactor: string
+}
+
+export type NativeCurrency = {
+  name: string
+  symbol: string
+  decimals: number
+  logoUri?: string
+}
+
+export type NetworkConfig = {
```

Comment:
This type already defines the values from the config service, to have as reference for next PRs